spec/rubocop/cop/style/return_nil_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Style::ReturnNil do
+  subject(:cop) { described_class.new(config) }
+
+  context 'when enforced style is `return`' do
+    let(:config) do
+      RuboCop::Config.new(
+        'Style/ReturnNil' => {
```

Comment:
I think the specs are failing because this doesn't contain the supported styles.